bitnami/cassandra/values.yaml

```diff
@@ -33,188 +69,56 @@ image:
-    - ReadWriteOnce
-  ## Persistent Volume size
-  ##
-  size: 8Gi
-  ## Commit Log Persistent Volume size
-  ##
-  # commitLogsize: 2Gi
-  ## The path the volume will be mounted at
-  ##
-  mountPath: /bitnami/cassandra
-  ## The path the volume will be mounted at
-  ##
-  # commitLogMountPath: /bitnami/cassandra/commitlog
-
-## Init containers parameters:
-## volumePermissions: Change the owner and group of the persistent volume mountpoint to runAsUser:fsGroup values from the securityContext section.
-##
-volumePermissions:
-  enabled: false
-  image:
-    registry: docker.io
-    repository: bitnami/bitnami-shell
-    tag: 10-debian-10-r112
-    pullPolicy: Always
-    ## Optionally specify an array of imagePullSecrets.
-    ## Secrets must be manually created in the namespace.
-    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
-    ##
-    # pullSecrets:
-    #   - myRegistryKeySecretName
-  ## Init container' resource requests and limits
-  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
-  ##
-  resources:
-    # We usually recommend not to specify default resources and to leave this as a conscious
-    # choice for the user. This also increases chances charts run on environments with little
-    # resources, such as Minikube. If you do want to specify resources, uncomment the following
-    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-    limits: {}
-    #   cpu: 100m
-    #   memory: 128Mi
-    requests: {}
-    #   cpu: 100m
-    #   memory: 128Mi
-  ## Init container Security Context
-  ## Note: the chown of the data folder is done to securityContext.runAsUser
-  ## and not the below volumePermissions.securityContext.runAsUser
-  ## When runAsUser is set to special value "auto", init container will try to chwon the
-  ## data folder to autodetermined user&group, using commands: `id -u`:`id -G | cut -d" " -f2`
-  ## "auto" is especially useful for OpenShift which has scc with dynamic userids (and 0 is not allowed).
-  ## You may want to use this volumePermissions.securityContext.runAsUser="auto" in combination with
-  ## pod securityContext.enabled=false and shmVolume.chmod.enabled=false
-  ##
-  securityContext:
-    runAsUser: 0
-
 ## Secret with keystore, keystore password, truststore, truststore password
 ## DEPRECATED. Use tls.existingSecret instead
 # tlsEncryptionSecretName:
 
-## ConfigMap with custom cassandra configuration files. This overrides any other Cassandra configuration set in the chart
+## Database credentials
+## @param dbUser.user Cassandra admin user
+## @param dbUser.forcePassword Force the user to provide a non
 ##
-# existingConfiguration:
+dbUser:
+  user: cassandra
+  forcePassword: false
+  # password:
+
+  ## Use an existing secrets which already stores your password data.
+  ## for backwards compatibility, existingSecret can be a simple string,
+  ## referencing the secret by name.
+  # existingSecret:
```

Comment:
We are not adding some variables like `password` or `existingSecret` to the README. Are we ok with that?

Reply:
Now they are added! Thanks